Lib/doctest.py

```diff
@@ -1607,12 +1610,22 @@ def check_output(self, want, got, optionflags):
         if got == want:
             return True
 
+        # Ignore case if flag
+        # Lowercase got and want
+        true_line = "True\n"
+        false_line = "False\n"
+        if (optionflags & IGNORE_CASE):
+            got = got.lower()
+            want = want.lower()
```

Comment:
Any reason not to use `got.casefold()` here?

Reply:
Good point.